attiny/attiny.ino

```diff
@@ -275,4 +348,13 @@ ISR(TIMER1_COMPA_vect) {
   }
 }
 
+bool isI2cReadReg(byte val) {
+  for (int i = 0; i < I2C_READ_REG_LEN; i++) {
+    if (i2cReadRegs[i] == val) {
+      return true;
+    }
+  }
```

Comment:
This is a bit overkill. If could just be `if (val == I2C_READ_BATTERY_VOLTAGE)  { ...`

Reply:
This was done so I can add more registers in the future to read. (see if the WDT was triggered for a restart is one idea)

Reply:
I figured that but for a small number of registers you could just have an if or switch.

Leaving it like this is fine though.